components/rmrk/Gallery/AvailableActions.vue

```diff
@@ -386,6 +386,7 @@ export default class AvailableActions extends mixins(
               `[${this.selectedAction}] ${this.nftId}`,
               notificationTypes.success
             )
+            this.$emit('change', this.selectedAction)
```

Comment:
hmm why do we need to emit `this.selectedAction` here though? you are not doing anything with the function parameter in parent